exercises/word-count/canonical-data.json

```diff
@@ -111,6 +111,15 @@
         "large": 2,
         "and": 1
       }
+    },
+    {
+      "description": "with multiple whitespace",
```

Comment:
A more helpful description would be nice.
What is it about multiple whitespace that makes it interesting to test?

"space surrounded by spaces not detected as a word"

Reply:
Thanks - a valid point. I fell into the trap of just imitating the overly brief descriptions from some of the prior test cases...